config/schedule.rb

```diff
@@ -35,3 +35,11 @@
 every :day, at: (ENV["FIRST_RENEWAL_EMAIL_REMINDER_DAILY_RUN_TIME"] || "1:05"), roles: [:db] do
   rake "email:renew_reminder:first:send"
 end
+
+# This is the registration exemptions exiry job which will collect all active
+# registration exemptions that have an expire date in the past and will set their
+# state to `expired`
+bulk_time = (ENV["EXPIRE_REGISTRATION_EXEMPTION_RUN_TIME"] || "20:05")
```

Comment:
Again, just for consistency we might as well keep this at the time we are going to run the job

```suggestion
bulk_time = (ENV["EXPIRE_REGISTRATION_EXEMPTION_RUN_TIME"] || "00:05")
```